osquery/tables/system/windows/shared_resources.cpp

```diff
@@ -0,0 +1,58 @@
+/*
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+
+#include <sstream>
```

Comment:
If you can use a `std::string` directly, instead of the `std::stringstream` then can be nixed.